src/style.css

```
@@ -0,0 +1,95 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "AvenirHevy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  --bordferColor: #e1e7ed;
+  --textColor: #3d4e61;
+  --shadow: rgba(61, 78, 97, 0.1);
+  --shadowHover: rgba(0, 0, 0, 0.25);
+  --shadowInputText: rgba(0, 0, 0, 0.25);
+  --gradientFocus: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  margin: 0 8px;
+}
+
+.search {
+  box-sizing: border-box;
+  outline: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: left;
+  height: 70px;
+  width: 100%;
+  margin: 20px 0 0;
+  border-radius: 4px;
+  border: 1px solid var(--bordferColor);
+  box-shadow: 0 1px 8px 0 var(--shadow);
+}
+
+.search--small {
+  height: 42px;
+  margin-bottom: 10px;
+}
+
+.search__icon {
+  box-sizing: border-box;
+  margin: 0 17px 0 26px;
+  height: 19px;
+  width: 19px;
+}
+
+.icon--small {
+  margin: 15px 10px 16px 12px;
+  height: 11px;
+  width: 11px;
+}
+
+.search__input {
+  box-sizing: border-box;
+  width: 100%;
+  border: none;
+  outline: none;
+  background-color: transparent;
+  padding: 0;
+
+  font-family: "AvenirHevy", sans-serif;
```

set it for `html` tag selector and let inputs inherit it 
```suggestion
  font-family: inherit;
```